packages/config/config/devices/0x024e/WIN-GDC-02.json

```diff
@@ -0,0 +1,323 @@
+				}
+			}
+		}
+	},
+	"paramInformation": {
+		"1": {
+			"$import": "templates/zconnect_template.json#relay_mode_selection",
+			"label": "Relay 1 Mode"
+		},
+		"2": {
+			"$import": "templates/zconnect_template.json#interval_1-65000",
+			"label": "Strobe Closed Interval",
+			"description": "Relay 1 and 2 closed contacts time interval in strobe mode.",
+			"defaultValue": 10
+		},
+		"3": {
+			"$import": "templates/zconnect_template.json#interval_1-65000",
+			"label": "Strobe Open Interval",
+			"description": "Relay 1 and 2 open contacts time interval in strobe mode.",
+			"defaultValue": 10
+		},
+		"4": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 Auto Off Delay",
+			"description": "Relay will turn off automatically after time specified in this parameter. After receiving command On Relay 1 will be switched Off automatically with predefined delay."
+		},
+		"5": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 On Delay",
+			"description": "Relay 1 will be switched On with predefined delay."
+		},
+		"6": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 1 Off Delay",
+			"description": "Relay 1 will be switched Off with predefined delay."
+		},
+		"7": {
+			"$import": "../templates/master_template.json#state_after_power_failure_prev_off",
+			"label": "Relay 1 Restore After Power Loss",
+			"description": "Restore to the last known state for relay 1 after power was interrupted."
+		},
+		"8": {
+			"label": "Broadband Command Action",
+			"description": "Broadband command turn all On/ turn all Off settings.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Accept broadband command turn all On/ turn all Off",
+					"value": 0
+				},
+				{
+					"label": "Ignore broadband command turn all On/ turn all Off",
+					"value": 1
+				},
+				{
+					"label": "Ignore broadband command turn all On, accept broadband command turn all Off",
+					"value": 2
+				},
+				{
+					"label": "Accept broadband command turn all On, ignore broadband command turn all Off",
+					"value": 3
+				}
+			]
+		},
+		"9": {
+			"$import": "../templates/master_template.json#base_enable_disable",
+			"label": "Hide Relay",
+			"description": "Hide relay 2 in controller interface"
+		},
+		"10": {
+			"$import": "templates/zconnect_template.json#relay_mode_selection",
+			"label": "Relay 2 Mode"
+		},
+		"11": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 2 Auto Off Delay",
+			"description": "Relay will turn off automatically after time specified in this parameter. After receiving command On Relay 2 will be switched Off automatically with predefined delay."
+		},
+		"12": {
+			"$import": "templates/zconnect_template.json#interval_1-65000_disable",
+			"label": "Relay 2 On Delay",
```

Comment:
Please make these consistent. Should Auto Off Delay and Auto On Delay. Change throughout.

Reply:
I think that it's correct as documented, Auto off delay is the delay before turning automatically off after being turned on, whereas On delay and Off delay is the delay between receiving the On command and actually turning On, and vis-a-vis for off.

Reply:
Are you sure? Usually it means like the switch has been off for 30 minutes so now turn on. Not actually a delay between the switch being activated and the on command.

Reply:
Yeh just tested it. It's a delay between the switch being activated and the relay changing state / the device reporting the state change.

Reply:
It's:
* on -> interval p12 -> device and relay switches on -> interval p11 -> auto off
* off -> interval p13 -> device and relay switches off